### PR TITLE
Refactor layout && make AdressDetails height adjustable

### PR DESCRIPTION
**Pull request by rizip1**
* the wrapper components in Public screen has often set zero height, what seems at least strange when viewed from console
* remove fixed position from header, but make it behave like it is (I try to avoid fix positioning if possible, let me know what think about it)

### CODE REVIEW

File: client/src/utils.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from 'react'
 import Loading from './components/Loading/Loading'
+import {AutoSizer} from 'react-virtualized'
```
Comment (mpinter) [inline client/src/utils.js]: the lib is huge - https://bundlephobia.com/result?p=react-virtualized@9.20.1

is it worth pulling it in just for this functionality ? can't the same be done by using correct markup & css ?

File: client/src/components/Public/Map/AddressDetail/AddressDetail.js [hunk 2]
```diff
@@ -21,16 +22,20 @@ type AddressDetailProps = {|
   entities: Array<Entity>,
   addressId: number,
   onClick: (e: Event) => void,
+  width: number,
+  height: number,
 |}
 
-const AddressDetail = ({entities, addressId, onClick}: AddressDetailProps) => (
+const DETAILS_HEADER_HEIGHT = 37
```
Comment (mpinter) [inline client/src/components/Public/Map/AddressDetail/AddressDetail.js]: keep constants in constants.js file

Comment (rizip1) [reply client/src/components/Public/Map/AddressDetail/AddressDetail.js]: Ok, but is it good practice to group unrelated constants (I mean just this case) that belong just to specific component and I cant really imagine this to be shared with the rest off app?

File: client/src/utils.js (hunk 1 above)
Comment (rizip1) [reply client/src/utils.js]: I can further investigate if the same can be easily achieved just with css/markup and will let you know.

**Pull request merged**